spec/bolt/cli_spec.rb

```diff
@@ -2698,123 +2698,6 @@ def stub_config(file_content = {})
     end
   end
 
-  describe 'project' do
```

Comment:
Hmm am I missing where these got replaced? They still seem useful.

Reply:
https://github.com/puppetlabs/bolt/pull/2211/files#diff-7fadf6af77342a81135f665eb01ac144c3eb19193ec8761e0143e723a85e9ae2R34-R122

Reply:
Oh I see. Those are unit tests though - we should have integration tests, either here or in the integration tests proper that cover these use cases.

Reply:
Those tests are 1:1 what was in cli_spec, granted they're rewritten to use `with_project`. I'm not sure what benefit having multiple integration tests gives us over the unit tests, since the CLI just instantiates a `ProjectManager` and calls `create`. I can add one that just makes sure a project name and `--modules` are passed along, but outside of that it'd just be duplicating tests.

Reply:
Ah, I missed that we were making a project around each example. You're right - I think adding a happy-path integration test would be good to ensure calling from the CLI works, but everything else is already tested.

Reply:
Updated with a happy path integration test. 👍